<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wally::qtfront::question_view_sub::MultipleChoicesRespMimic</class>
 <widget class="QGroupBox" name="wally::qtfront::question_view_sub::MultipleChoicesRespMimic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Multiple Choices Response</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="layout_choices"/>
   </item>
   <item>
    <widget class="QPushButton" name="btn_addChoice">
     <property name="text">
      <string>Add Choice</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_del">
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
